holoviews/util/__init__.py

```diff
@@ -2,13 +2,42 @@
 
 import param
 
-from .core import DynamicMap, HoloMap, ViewableElement
-from .core.operation import Operation
-from .core.util import Aliases  # noqa (API import)
-from .core.operation import OperationCallable
-from .core.spaces import Callable
-from .core import util
-from .streams import Stream
+from ..core import DynamicMap, HoloMap, Dimensioned, ViewableElement, StoreOptions, Store
+from ..core.options import options_policy
+from ..core.operation import Operation
+from ..core.util import Aliases  # noqa (API import)
+from ..core.operation import OperationCallable
+from ..core.spaces import Callable
+from ..core import util
+from ..streams import Stream
+from .settings import OutputSettings
+
+
+Store.output_settings = OutputSettings
+
+# Needs same validation behavior!
+def opts(options, obj=None):
+    from .parser import OptsSpec
+    if obj is None:
+        with options_policy(skip_invalid=True, warn_on_skip=False):
+            StoreOptions.apply_customizations(OptsSpec.parse(options), Store.options())
+    elif not isinstance(obj, Dimensioned):
+        return obj
+    else:
+        return StoreOptions.set_options(obj, OptsSpec.parse(options))
+
+
+def output(line=None, obj=None, **options):
+    """
+    Cell magic version acts as a no-op.
+    """
+    if obj is not None:
+        return obj
```

Comment:
Could you outline how you can imagine that working? Even if I set the options state temporarily for an expression, it will have no effect unless that expression has some sort of side-effect, namely output. The cell magic resets everything after display and this concept of 'display' doesn't exist outside the notebook.

If you do have ideas about this, I would like to hear them as I would like these utilities to correspond to the magics as closely as possible...